ft_calloc.c

```diff
@@ -0,0 +1,25 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_calloc.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: ldamiens <ldamiens@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2021/11/06 16:44:12 by ldamiens          #+#    #+#             */
+/*   Updated: 2021/11/11 16:10:37 by ldamiens         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <stdlib.h>
+#include "libft.h"
+
+void	*ft_calloc(size_t count, size_t size)
+{
+	char	*ptr;
+
+	ptr = malloc(count * size * sizeof(char));
```

Comment:
`sizeof(char)` is guaranteed to always be 1 and either way it does not make much sense in this context since the size is already given by `size` (also it is not the same as the bzero below)
```suggestion
	ptr = malloc(count * size);
```